src/main/kotlin/no/nav/syfo/melding/api/ReturAvLegeerklaringRequestDTO.kt

```diff
@@ -0,0 +1,34 @@
+package no.nav.syfo.melding.api
+
+import no.nav.syfo.melding.domain.*
+import java.time.OffsetDateTime
+import java.util.*
+
+data class ReturAvLegeerklaringRequestDTO(
+    val document: List<DocumentComponentDTO>,
+)
+
+fun ReturAvLegeerklaringRequestDTO.toMeldingTilBehandler(
+    opprinneligUtgaendeForesporsel: MeldingTilBehandler,
+    innkommendeLegeerklaringMelding: MeldingFraBehandler,
+    veilederIdent: String,
+): MeldingTilBehandler {
+    val now = OffsetDateTime.now()
+    return MeldingTilBehandler(
+        uuid = UUID.randomUUID(),
+        createdAt = now,
+        type = MeldingType.HENVENDELSE_RETUR_LEGEERKLARING,
+        conversationRef = opprinneligUtgaendeForesporsel.conversationRef,
+        parentRef = innkommendeLegeerklaringMelding.uuid,
+        tidspunkt = now,
+        arbeidstakerPersonIdent = opprinneligUtgaendeForesporsel.arbeidstakerPersonIdent,
+        behandlerPersonIdent = opprinneligUtgaendeForesporsel.behandlerPersonIdent,
+        behandlerNavn = opprinneligUtgaendeForesporsel.behandlerNavn,
+        behandlerRef = opprinneligUtgaendeForesporsel.behandlerRef,
```

Comment:
Noe å tenke på her: Hva om vi sender ut en forespørsel til en behandler, men så svarer en annen. Da vil vi kanskje heller bruke `behandlerNavn,` `behandlerRef` og `behandlerPersonIdent` fra det innkomne svaret? Problemet er at en `MeldingFraBehandler` mangler `behandlerRef`-feltet, så må få hentet ut det på en eller annen måte i så fall.

Reply:
Vi beholder det slik, se avklaring: https://nav-it.slack.com/archives/C04M11803C1/p1692278094467089